db/seeds.rb

```diff
@@ -199,4 +199,4 @@
 Donation.create(amount: 2000, candidate_issue_id: candidate_issue1.id, user_id: user3.id, order_id: order15.id)
 
 Donation.create(amount: 1000, candidate_issue_id: candidate_issue2.id, user_id: user3.id, order_id: order15.id)
-puts "Orders and Donations created"
+puts "Orders and Donations created"
```

Comment:
Final newline missing.<br>Do not write to stdout. Use Rails' logger if you want to log.